packages/esm-patient-clinical-view-app/src/clinical-encounter/summary/out-patient-summary/patient-medical-history.component.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { formatDate, parseDate, useConfig } from '@openmrs/esm-framework';
+import {
+  Surgical_History_UUID,
+  Accident_Trauma_UUID,
+  Blood_Transfusion_UUID,
+  Chronic_Disease_UUID,
```

Comment:
I would recomment to have this in snake case and upper case e.g 
```suggestion
  SURGICAL_HISTORY_UUID,
  ACCIDENT_TRAUMA_UUID,
  BLOOD_TRANSFUSION_UUID,
  CHRONIC_DISEASE_UUID,
```